content/en/docs/concepts/storage/volume-pvc-datasource.md

```diff
@@ -32,7 +32,7 @@ The {{< glossary_tooltip text="CSI" term_id="csi" >}} Volume Cloning feature add
 
 A Clone is defined as a duplicate of an existing Kubernetes Volume that can be consumed as any standard Volume would be.  The only difference is that upon provisioning, rather than creating a "new" empty Volume, the back end device creates an exact duplicate of the specified Volume.
 
-The implementation of cloning, from the perspective of the Kubernetes API simply adds the ability to specify an existing unbound PVC as a dataSource during new pvc creation. 
+The implementation of cloning, from the perspective of the Kubernetes API simply adds the ability to specify an existing unbound PVC as a dataSource during new pvc creation.
```

Comment:
The PVC example below doesn't specify a storageclass, but we made it required right?

Reply:
StorageClass is not actually required, it's just required that the source and destination are in the same storageClass.  This can either be specified via storageClassName or by having a default storageClass.

I updated the doc to try and clarify that.